backend/amd64/simd_selection.ml

```diff
@@ -49,6 +54,14 @@ let float_condition_of_int = function
   | 7 -> ORDf
   | i -> Misc.fatal_errorf "Invalid float condition immediate: %d" i
 
+let float_rounding_of_int = function
```

Comment:
I suggest changing this convention, even though it matches the constants used in the manual. The way I see it: the name of the built-in is more informative than the opcode. Same should hold for arguments. 
It would be much better to have the argument spelled out in the name of the instrinsic, similar to the way it is handled in `seleciton.ml` for `caml_float_iround_half_to_even_unboxed`. Same for `float_condition_of_int` above. It would for example ensure that the const is untagged (doesn't help for other uses of `extract_constant`).

Reply:
Agreed - they're now defined in `ocaml_intrinsics` and not exposed elsewhere, so I think it's fine.